src/components/shared/Switcher.js

```diff
@@ -15,18 +15,19 @@ const SwitcherItem = styled.p`
   margin: 0;
   padding: 3% 0;
   color: ${props => (props.active ? props.theme.colors.primaryDark : '#333')};
-  border-bottom: ${props => (props.active ? '2px solid' + props.theme.colors.primaryDark : 'none')}
-
+  border-bottom: ${props =>
+    props.active ? '2px solid' + props.theme.colors.primaryDark : 'none'};
 `
 
 class Switcher extends React.Component {
   render() {
     return (
       <SwitcherWrapper>
-        {this.props.links.map(link => {
+        {this.props.links.map((link, i) => {
```

Comment:
When only returning a component inside an arrow function, it can be simplified like so:
```javascript
{this.props.links.map((link, i) => (
  <Component />
))}
```
No need for extra curly braces and a `return` statement.